packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [3.6.1]
+* Updated Github workflow files generated with the `github` cicd flag
```

Comment:
Let's include the CI fix as well

Reply:
This should be addressed. I've created a ticket to increase the code coverage for the project(s) generated with rx_bloc_cli and it can be found [here](https://github.com/Prime-Holding/rx_bloc/issues/721).